src/components/LeaderboardPositionCard/LeaderboardPositionCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Card from 'components/Card/Card';
+
+import { ReactComponent as UpArrowIcon } from './ArrowUp.svg';
+import { ReactComponent as DownArrowIcon } from './ArrowDown.svg';
+
+import styles from './LeaderboardPositionCard.module.css';
+
+type LeaderboardUpDownProps = {
+  direction: 'up' | 'down';
+  points: number;
+};
+
+function LeaderboardUpDown({ direction, points }: LeaderboardUpDownProps) {
+  return (
+    <div className={`${styles['up-down']} ${styles[direction]}`}>
+      {direction === 'up' ? <UpArrowIcon /> : <DownArrowIcon />}
+      <span>{`${points}pts`}</span>
+    </div>
+  );
+}
+
+type Props = {
+  className?: string;
+};
+
+export default function LeaderboardPositionCard({ className }: Props) {
```

Comment:
I think this name could be confused with the Leaderboard Cards we will need for the complete leaderboard.
Maybe a different name to state this is specifically for the dashboard could be useful
<img width="610" alt="image" src="https://github.com/Code4UHub/client/assets/60989884/80350829-8eef-4de0-87f6-0aa100348375">